pkgs/games/factorio/default.nix

```diff
@@ -17,7 +17,7 @@ let
   variants = {
     x64 = {
       url = "https://www.factorio.com/get-download/${version}/alpha/linux64";
-      sha256 = "1xpzrx3q678519qgjl92fxn3qv55hd188x9jp6dcfk2ljhi1gmqk";
+      sha256 = "1dc53swnlbz23czw8v3ldl4ayp7118qyja4vh1z5dsd871swz9qg";
     };
 
     x32 = {
```

Comment:
> That would be a separate derivation, though it could probably share the config stuff. Note that, although it's still unfree, http://www.factorio.com/download-headless/stable allows you to download the headless version without having a username/password—that's certainly deliberate.

Awesome, will try to make a package for this later today.

> Also feel free to drop by the #nixos IRC channel if you want to talk to me. I think I'm going to see if I can get this working on OS X as well...

Will do. I'm pretty sure I'll have some more questions when building the headless package. An OS X version sounds great, happy to test it once it's ready.